### ISSUE TITLE
Phoenix.LiveViewTest.Dom.find_session/1 is [] for large sessions

### ISSUE THREAD

**Issue by jjcarstens** [phoenixframework/phoenix_live_view#201]
### Environment

* Elixir version (elixir -v): elixir 1.8.1 (compiled with Erlang/OTP 21)
* Phoenix version (mix deps): 1.4.3
* NodeJS version (node -v): v8.11.3
* NPM version (npm -v): 5.6.0
* Operating system: MacOS 10.14.4

### Actual behavior

If the session data byte_size is ~ > 4600, then the `Phoenix.LiveView.DOM.find_sessions/1` returns an empty list instead of finding the session. Subsequently. this can break tests that use `Phoenix.LiveViewTest.mount/3` or `Phoenix.LiveViewTest.mount_disconnected`.

It seems to be related to this [recent change to the regex for finding the session](https://github.com/phoenixframework/phoenix_live_view/commit/4bec625a2d426c36531b2bed3a734eeac9b5aff8#diff-3ddc681ae3575d08d4d65073fc1cff95R42).

I haven't been able to pinpoint the exact byte size that breaks because it seems to depend upon what is in the same `div` as `data-phx-session` is so its likely related to Regex handling all the reverse lookups.

You can reproduce with the following steps:

```elixir
too_much_data = Enum.map(1..4458, fn _ -> "t" end) |> Enum.join
ok_data = Enum.map(1..4457, fn _ -> "t" end) |> Enum.join

bad_html = """
<div data-phx-session="#{too_much_data}" id="howdy"></div>
"""

ok_html = """
<div data-phx-session="#{ok_data}" id="howdy"><div>
"""

Phoenix.LiveViewTest.DOM.find_session(bad_html) == []
Phoenix.LiveViewTest.DOM.find_session(ok_html) == [{ok_data, nil, "howdy"}]
```

### Expected behavior

`Phoenix.LiveViewTest.DOM.find_sessions/1` should be able to find any size session in the html text.

**Comment by jakedjohnson** [comment]
I was able to get this working with a couple of different variations to the regex sigil in `Phoenix.LiveViewTest.DOM.find_sessions/1`.

Both of the following successfully captured larger sessions for me:
- `~r/<div\s[^>]+data-phx-session="([^"]+)[^>]+data-phx-static="([^"]+)[^>]+id="([^"]+)|<[^>]+data-phx-session="([^"]+)[^>]+id="([^"]+)/`
- `~r/data-phx-session="([^"]+)[^>]+data-phx-static="([^"]+)[^>]+id="([^"]+)|<[^>]+data-phx-session="([^"]+)[^>]+id="([^"]+)/`

And another variation from proposed fix PR #197:
- `~r/<[^>]+data-phx-session="([^"]++)[^>]+data-phx-static="([^"]+)[^>]+id="([^"]+)|<[^>]+data-phx-session="([^"]++)[^>]+id="([^"]+)/`

**Comment by jjcarstens** [comment]
Ah man, I really should have looked at the PRs before having to trudge through this myself! Thanks @jakedjohnson!
